client/src/components/ui/signup/regex.ts

```diff
@@ -0,0 +1,7 @@
+// regex.ts
+export const emailRegex = new RegExp(
+  "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$",
+);
+export const mobileRegex = new RegExp("^[0-9]{10}$");
+
+export const userNameRegex = new RegExp("^[w.@+-]+$");
```

Comment:
Why is there a file for just regex? Should be in the file that they are used in

Reply:
I'm thinking, in the future, if people want to validate more forms, they can extract regular expressions from this file. I actually want to move this to another location instead of keeping it separately in the `components/ui/signup` folder.

Reply:
Hm.. it doesn't make sense to separate this into its own file, regex strings are nothing more than just variables and we don't usually separate variables out into its own file (unless the variable is a global constant/configuration that affects the whole project)